src/toast/hoc/toast.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react'
 
 export const addDelay = (prevProps, props, defaultDelay, timer) => {
-  const { print, code, remove, delay } = props
-  if (prevProps.code !== code && print) {
+  const { print, title, type, remove, delay } = props
```

Comment:
I asked myself, is `print` property necessary now ? 
Because as I understand the API, there is only one **toast** in the store at the time, which is the one to be printed ?